### PR TITLE
games-strategy/colobot: new package

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

**Comment by reagentoo** [comment]
I think it is advisable to use `colobot-music_{flac,ogg}_0.1.11-alpha.tar.gz` in the live ebuild too.

File: games-strategy/colobot-data/colobot-data-0.1.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils cmake-utils
+
+DESCRIPTION="Data package for colobot (Colonize with Bots)"
+HOMEPAGE="https://colobot.info/"
+
+SRC_URI="
+	https://github.com/colobot/colobot-data/archive/colobot-gold-${PV}-alpha.zip -> ${P}.zip
```
Comment (Zlogene) [inline games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: put `app-arch/unzip` into `DEPEND`

File: games-strategy/colobot-data/colobot-data-0.1.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils cmake-utils
```
Comment (Zlogene) [inline games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: do you really need `eutils` ?

File: games-strategy/colobot-data/colobot-data-0.1.11.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+DESCRIPTION="Data package for colobot (Colonize with Bots)"
+HOMEPAGE="https://colobot.info/"
+
+SRC_URI="
+	https://github.com/colobot/colobot-data/archive/colobot-gold-${PV}-alpha.zip -> ${P}.zip
+	music_ogg? ( https://colobot.info/files/music/colobot-music_ogg_${PV}-alpha.tar.gz -> ${P}-music-ogg.tar.gz )
+	music_flac_convert? ( https://colobot.info/files/music/colobot-music_flac_${PV}-alpha.tar.gz -> ${P}-music-flac.tar.gz )
+"
+KEYWORDS="~amd64"
+DEPEND="music_flac_convert? ( media-sound/vorbis-tools )"
```
Comment (Zlogene) [inline games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: variabled order is broken, why should `DEPEND` be here

Comment (soredake) [reply games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: It converts flac music to ogg format with oggenc from vorbis-tools

File: games-strategy/colobot-data/colobot-data-0.1.11.ebuild (hunk 1 above)
Comment (soredake) [reply games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: done

File: games-strategy/colobot-data/colobot-data-0.1.11.ebuild (hunk 2 above)
Comment (soredake) [reply games-strategy/colobot-data/colobot-data-0.1.11.ebuild]: removed

**Comment by reagentoo** [comment]
@SoapGentoo 
```
+DIST colobot-data-0.1.11-music-flac.tar.gz 298829480
+DIST colobot-data-9999-music-flac.tar.gz 298829480
```
298829480=298829480
Nice work! Make Gentoo mirror more fat.

**Comment by SoapGentoo** [comment]
@reagentoo 
1. Thanks for the passive-aggressiveness, and valuing my volunteer work.
2. Space is cheap in 2018
3. Ok, so you want to take over the responsibility for maintaining the -9999 -> colobot-data-0.1.11-music-flac.tar.gz mapping and checking that the latest flac tarball is the same?
4. If this bothers you at a very deep level, send in a PR.

**Pull request closed without merging**